File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/C/program1.c

#include <stdio.h>

int main () {

	int n, flag = 0;
	printf("Enter Number : ");
	scanf("%d", &n);

	if(n < 0)
		exit(0);
	for(int i = 2; i <= n/2; i++){
		
		if(n % i == 0 ){
			printf("Not prime Number\n");
			flag = 1;
			break;
		}
	}

	if(flag == 0){
		printf("Prime Number\n");
	}
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Python/program1.py
try:
	n = int(input("Enter Number : "))
except ValueError as e:
	exit(0)

if(n < 0):
	exit(0)
flag = 0
for i in range(2, n//2 + 1):	
	if(n % i == 0 ):
		print("Not prime Number")
		flag = 1
		break
if(flag == 0):
	print("Prime Number")
	

